script/Arcueid/Day 8/08_00_ARC08_6.txt

```diff
@@ -48,7 +48,7 @@ You promised you'd be here over ten minutes ago!"
 -- Page 291, Offset 20398.
 -- ……10分もって……。そりゃあ、屋敷を出たのが10時前だったから、全力で走ってきても時間通りに到着はできない。
 Ten minutes...
- I mean, despite the fact that I left the mansion before 10 o'clock,
+ I mean, despite the fact that I left the mansion before ten o'clock,
```

Comment:
Should I change it for temperatures and measurements too? since there are two instances of that.